wikiclue/src/components/dailyEnd.svelte

```diff
@@ -1,44 +1,55 @@
 <script lang="ts">
-    import Overlay from "./overlay.svelte";
-	import { writable } from "svelte/store";
-    export let levelOpen = false;
+	import { goto } from '$app/navigation';
+	import Overlay from './overlay.svelte';
+	import { writable } from 'svelte/store';
+	export let levelOpen = false;
 
-    let statistics = {'Played' : 24, 'Win %': '85%', 'Current Streak': 3, 'Max Streak': 7};
-    let distribution = [4, 1, 3, 3, 4, 1];
-    let max = 4;
-    const isOverlayOpen = writable(false);
+	export let statistics = { Played: 24, 'Win %': '85%', 'Current Streak': 3, 'Max Streak': 7 };
```

Comment:
Why is only played not a string but the rest are?

Reply:
My linter for some reason didn't like quotes around it, but this has been adjusted